modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -39,16 +44,17 @@ Note: When multiple partitions are built parallel, it needs more RAM and mandate
 
 === Failovers 
 
-During failover of Search service nodes, there is no partition movement, and hence the failover-rebalance is instantaneous. Search service promotes the replica index partitions to primary so that those serve the live cluster traffic instantly. Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. 
+During failover of FTS service nodes, there is no partition movement, and hence the failover-rebalance is instantaneous. Search service promotes the replica index partitions to primary so that those serve the live cluster traffic instantly. Users could use failover and recovery rebalance for applying patches or upgrading software/hardware for a shorter duration. 
  
 === Things to keep track of during a rebalance
 
-During a strict recovery rebalance operation (no extra node additions/removals), the index partitions residing on the recovered node would be reused. And this ensures a quick recovery rebalance operation.
+A failed over node can be recovered: added back into the cluster from which it was failed over through the rebalance operation. 
+The index partitions residing on the recovered node would be reused during a recovery rebalance operation with no extra node additions/removals. And this ensures a quick recovery rebalance operation.
```

Comment:
And this ensures a quick recovery rebalance operation. -> This speeds up the recovery rebalance operation.

Reply:
👍

Reply:
Updated